pages/playground.js

```diff
@@ -75,7 +75,8 @@ export default function Playground() {
       if (gitUsername && profileJson && handleValidateJson()) {
         setErrMsg("");
         let actualJson = { username: gitUsername, ...JSON.parse(profileJson) };
-        delete actualJson.testimonials;
+        actualJson.testimonials = actualJson.testimonials || [];
+        actualJson.socials = actualJson.socials || [];

```

Comment:
Please can you explain these changes?

Reply:
When you press Preview with empty JSON or JSON without testimonials and socials are given an undefined error on UserPage.js on UserProfile.js file respectively

Reply:
Oh I see, yes you are correct. I think I fixed `UserProfile` for the empty socials. But we can leave these in for now as extra checks